src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -3,668 +3,792 @@
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
-#! format: noindent
```

Comment:
Sadly, all of our source files will be indented because of this...

Reply:
There seems to be no way around it. I tried
```julia
# runic: off
@muladd begin
# runic: on
```
but this just disables formatting for the whole file.

Reply:
https://github.com/fredrikekre/Runic.jl/issues/60

Reply:
See the reply in the linked issue. That's your decision now. We don't use this "wrapping full files in @muladd" thing in TrixiParticles.jl, so I don't really care about this indent.